Makefile

```
@@ -0,0 +1,15 @@
+clean_translations_temp_directory:
+	rm -rf I18N/I18N/
+
+
+pull_translations:
+	make clean_translations_temp_directory
+	atlas pull $(ATLAS_OPTIONS) translations/openedx-app-ios/I18N/I18N/:I18N/I18N/
+	python i18n_scripts/split_translation_files.py
+	make clean_translations_temp_directory
+
+
+combine_translations:
+	make clean_translations_temp_directory
+	pip3 install localizable==0.1.2
+	python3 i18n_scripts/combine_translations_files.py
```

Please add the following lines:

```suggestion
create_virtual_env:
	rm -rf .venv
	python3 -m venv .venv
	. .venv/bin/activate && pip install localizable==0.1.2
```

combine_translations: create_virtual_env
	make clean_translations_temp_directory
	. .venv/bin/activate && python3 i18n_scripts/combine_translations_files.py
```

---

done for now